miio/integrations/roborock/vacuum/vacuum.py

```diff
@@ -153,6 +156,7 @@ def __init__(
         self._map_enum_cache = None
         self._status_helper = UpdateHelper(self.vacuum_status)
         self._status_helper.add_update_method("consumables", self.consumable_status)
+        self._status_helper.add_update_method("connectivity", self.connectivity_status)
```

Comment:
Please move this into its own PR, as that will require some more thought and the rest of the changes are good to go.

One way would be to move the "helper" and exposing common sensors like wifi signal in the base class. I'm a bit hesitant on adding more I/O to the main update cycle for two reasons: (1) not all devices support this (none of I have does, but they are _old_ so maybe all newer ones do) and (2) devices that are disconnected from the cloud are known to fail to respond to info properly.

It could be that all of this is irrelevant with modern devices and firmware versions, but I'd like to avoid potentially breaking working setups, including my own, and have this in a separate PR to allow merging this one.

Reply:
Done